scripts/updates/check-updates.sh

```diff
@@ -49,7 +49,7 @@ if [ -f "${BASEDIR}/github-projects.txt" ]; then
 			latest_version=$(grep -oP "$version_regexp" <<< "$latest_version" || true)
 		fi
 
-		if [ -z "$latest_version" ]; then
+		if [ -z "$latest_version" ] | [ "$latest_version" = "null" ]; then
```

Comment:
Did you mean `||`?

Reply:
single `|` is expected.

Reply:
Thanks, single `|` is worked but double `||` is more suitable.

Reply:
There is a big difference between `|` and `||`.